docs/docs/examples/cookbooks/airtrain.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.            if not isinstance(ev, CompletedDocumentRetrievalEvent):
</pre><p>It's actually not necessary to check the type of event. Under the hood, inspection is being done on the step's to determine its accepted events.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/16332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>removed this</p><p> </p>